syntax/arrai.wbnf

```diff
@@ -30,7 +30,7 @@ expr   -> C* amp="&"* @ C* arrow=(
         | C* "{" C* set=(elt=@:",",?) "}" C*
         | C* "{" C* dict=((key=@ ":" value=@):",",?) "}" C*
         | C* cond=("cond" "(" (key=@ ":" value=@):",",? ("*" ":" f=expr ","?)? ")") C*
-        | C* cond=(("(" control_var=expr ")" | IDENT)? C* "cond" "(" (key=@ ":" value=@):",",? ("*" ":" f=expr ","?)? ")") C*
+        | C* cond=(("(" control_var=expr ")" | IDENT)? C* "cond" "(" (key=(("(" @:"," ")") | @) ":" value=@):",",? ("*" ":" f=expr ","?)? ")") C*
```

Comment:
This should be much lower precedence. Put it right above or below the original `if...else` construct. Just the `x cond (...)` syntax, mind you. The `cond (...)` form can stay where it is.